storybook/manager-head.html

```diff
@@ -3,6 +3,7 @@
 		const PREVIOUSLY_EXPERIMENTAL_COMPONENTS = [
 			'alignmentmatrixcontrol',
 			'customselectcontrol-v2',
+			'dimensioncontrol',
```

Comment:
Is this redirect going to work? Usually we change from "experimental" to "stable", but in this instance we're changing to "deprecated" — would this affect the new URL?

Reply:
Unless I misunderstand, this is unnecessary.

Reply:
Good question. For the components moved to the Deprecated section, we're explicitly [setting an id](https://github.com/WordPress/gutenberg/pull/64951/files#diff-8df0249b1ef78119af90360b4bb0631dec3514a889e2297258709cb0976f32acR24) in the story meta so it won't include the word "deprecated" in the URL.